examples/bayesian_regression.py

```diff
@@ -51,10 +52,11 @@ def forward(self, x):
 def model(data):
     # Create unit normal priors over the parameters
     mu = torch.zeros(1, p).type_as(data)
-    sigma = torch.ones(1, p).type_as(data)
+    sigma = 2 * torch.ones(1, p).type_as(data)
```

Comment:
Would you mind modernizing all of this initialization code to avoid `.type_as()` (since `.type_as()` does not work on multi-gpu systems):
```py
mu = data.new_zeros(1, p)
sigma = 2 * data.new_ones(1, p)
bias_mu = data.new_zeros(1)
bias_sigma = 2 * data.new_ones(1)
```

Reply:
I guess we can fix these in a later PR since many examples need to be fixed.

Reply:
i was actually in the process of doing that but the mvn pr broke my local env since i think it needs a newer pt version so i couldnt test cuda. will open another pr with this fix